tmt/steps/provision/local.py

```diff
@@ -268,6 +284,17 @@ def go(self, *, logger: Optional[tmt.log.Logger] = None) -> None:
         if data.hardware and data.hardware.constraint:
             self.warn("The 'local' provision plugin does not support hardware requirements.")
 
+        assert isinstance(self.parent, tmt.steps.provision.Provision)
+        assert self.parent.plan.my_run is not None
+        workdir = self.parent.plan.my_run.workdir
+        assert workdir is not None
+
+        if os.environ.get("TMT_SCRIPTS_DIR"):
+            self.warn(
+                "The TMT_SCRIPTS_DIR env is not supported in 'local' provision, "
+                f"the default scripts path {workdir / Path('scripts')} is used instead."
```

Comment:
```suggestion
                f"the default scripts path {workdir.absolute() / 'scripts'} is used instead."
```

Reply:
Updated ! Works nice as well
`warn: The TMT_SCRIPTS_DIR env is not supported in 'local' provision, the default scripts path /var/tmp/tmt/run-012/scripts is used instead.`

Reply:
I've updated to use `self._guest.scripts_path` instead, less duplication.